sdk/ml/azure-ai-ml/tests/automl_job/e2etests/test_automl_image_segmentation.py

```diff
@@ -75,6 +75,9 @@ def test_image_segmentation_run(self, image_segmentation_dataset: Tuple[Input, I
         training_data = Input(type=AssetTypes.MLTABLE, path=train_path)
         validation_data = Input(type=AssetTypes.MLTABLE, path=val_path)
 
+        properties = get_automl_job_properties()
```

Comment:
by doing this, aren't we only launching jobs with the components?

Reply:
That's correct, we are only launching jobs with components. Would it be better to keep the runtime tests and duplicate for components even if the runtime component is tested?

Reply:
but this PR is removing the tests for the current e2e workflow, no? we want to add component tests on top of existing ones, not to replace them ...

Reply:
What are your thoughts on the [dsl tests](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/tests/dsl/e2etests/test_automl_dsl_pipeline.py)? The run is kicked off and immediately cancelled for the rest object can be inspected, so the behavior is the same with or without the feature flag.

Reply:
Discussed offline with Phani, we decided that because dsl tests don't actually submit runs that we shouldn't duplicate them.